Formula/apt-dater.rb

```diff
@@ -1,8 +1,11 @@
 class AptDater < Formula
   desc "Manage package updates on remote hosts using SSH"
   homepage "https://www.ibh.de/apt-dater/"
-  url "https://downloads.sourceforge.net/project/apt-dater/apt-dater/0.9.0/apt-dater-0.9.0.tar.gz"
-  sha256 "10b8335c156dabae249aa071cf8689900fae325c2e9540e36a840a2a77d3eeb4"
+  url "https://github.com/DE-IBH/apt-dater.git",
+      :tag => "v1.0.3",
+      :revision => "fdad3b43e50bc3582e391d6feabae150e133b3b3"
```

Comment:
Maybe can use https://github.com/DE-IBH/apt-dater/releases, but why though? I thought git repo is better for Homebrew.